packages/postcss-ordered-values/src/index.js

```diff
@@ -22,13 +23,19 @@ const borderRules = {
   'border-left': border,
 };
 
+const columnRules = {
+  'column-rule': border,
```

Comment:
better create separate function, or rename `border` to `borderAndColumn`

Reply:
cool 👍